server/controllers/authController.ts

```diff
@@ -66,13 +66,18 @@ export const getToken = catchAsync(
   async (req: Request, res: Response) => {
     const tokenRequest = {
       code: req.body.code as string,
-      scopes: ['user.read'],
+      scopes: ['openid'],
+      nonce: 123,
```

Comment:
Out of curiosity, what is `nonce` and why the value of that is `123`?

Reply:
Nonce is a field that originally is generated by MS to identify token as token only for MS Graph (so without possibility to validate). I have to add this field manually with some value to overwrite auto-generated nonce field to indicate that this token is not only for MS Graph